build/__phpVersions.sh

```diff
@@ -10,6 +10,8 @@ COMPOSER1_10_VERSION='1.10.19'
 COMPOSER1_10_SETUP_SHA384='756890a4488ce9024fc62c56153228907f1545c228516cbf63f885e036d37e9a59d27d63f46af1d4d07ee0f76181c7d3'
 COMPOSER2_0_VERSION='2.0.8'
 COMPOSER2_0_SETUP_SHA384='756890a4488ce9024fc62c56153228907f1545c228516cbf63f885e036d37e9a59d27d63f46af1d4d07ee0f76181c7d3'
+COMPOSER2_2_VERSION='2.2.9'
+COMPOSER2_2_SETUP_SHA384='756890a4488ce9024fc62c56153228907f1545c228516cbf63f885e036d37e9a59d27d63f46af1d4d07ee0f76181c7d3'
 PHP81_VERSION='8.1.0'
```

Comment:
I believe I mentioned a version update for PHP as well to 8.1.4

Reply:
I see `platforms/php/versionsToBuild.txt` correctly updated, maybe we need to update here as well?

Reply:
Same here. These files not updated since these versions are for runtimes.